src/schema/core.yaml

```diff
@@ -1709,16 +1730,21 @@ slots:
       - "key set to false due to rare collisions: Pletnev I, Erin A, McNaught A, Blinov K, Tchekhovskoi D, Heller S (2012) InChIKey collision resistance: an experimental testing. J Cheminform. 4:12"
   inchi:
     range: string
-  peptide_sequence_count:
+  
+  unique_peptide_count:
     description: count of peptide sequences grouped to the best_protein
     range: integer
+
   protein_spectral_count:
     description: sum of filter passing MS2 spectra associated with the best protein within a given LC-MS/MS data file
```

Comment:
sum of filter passing MS2 spectra associated with a given razor protein within a given LC-MS/MS data file